packages/dashboard/src/components/Recommendation/__snapshots__/Recommendation.test.tsx.snap

```diff
@@ -277,7 +277,7 @@ Array [
               <p
                 style={
                   Object {
-                    "fontSize": 28,
+                    "fontSize": undefined,
```

Comment:
is this expected?

Reply:
yes, since the jest does not evaluate the css at very beginning